index.html

```diff
@@ -178,30 +190,47 @@
         rules in this section apply.
       </p>
       <pre class="idl">
+        enum CaptureFacingMode { "user", "environment" };
+
         partial interface HTMLInputElement {
-            [CEReactions] attribute boolean capture;
+            [CEReactions] attribute CaptureFacingMode capture;
         };
       </pre>
       <div>
         <p>
-          The <code><a>capture</a></code> attribute is a <a>boolean
-          attribute</a> that, if specified, indicates that the capture of media
-          directly from the device's environment using a <a>media capture
-          mechanism</a> is preferred.
+          The <dfn data-dfn-for="CaptureFacingMode">CaptureFacingMode</dfn>
+          enumeration is used to express the <a>preferred facing mode</a>. The
+          semantics of its keywords mirror the similarly named keywords defined
+          in <a><code>VideoFacingModeEnum</code></a>.
```

Comment:
The VideoFacingModeEnum is video-specific, so thought adding an indirection would be justified, as it'd allow us to have control over the semantics as needed.

That said, I don't feel strongly about this, we could reuse VideoFacingMode as a hint for audio or image capture too given the enum name itself is not web-facing.